src/lib.rs

```diff
@@ -1,5 +1,5 @@
 #![deny(missing_docs)]
-#![deny(warnings)]
+// #![deny(warnings)]
```

Comment:
```suggestion
```

Reply:
Oops! Good catch, you may want to clean this up now that this landed.

Reply:
Nah, it's better to enforce it in CI via rustflags, so this removal is something I'd want to do myself. Otherwise, newer versions of rust won't be able to compile toml_edit if it produces a warning in the future.